cibyl/sources/zuuld/tools/yaml.py

```diff
@@ -17,50 +17,96 @@
 from dataclasses import dataclass, field
 from typing import Iterable, Optional
 
+from overrides import overrides
+
 from cibyl.sources.zuuld.models.job import Job
 from kernel.tools.files import FileSearchFactory
-from kernel.tools.fs import Dir, File, KnownDirs, cd
+from kernel.tools.fs import Dir, File
 from kernel.tools.json import Draft7ValidatorFactory
-from kernel.tools.yaml import YAMLArray, YAMLError, YAMLFile
+from kernel.tools.urls import URL
+from kernel.tools.yaml import (YAMLArray, YAMLError, YAMLFile,
+                               YAMLValidatorFactory)
 
 LOG = logging.getLogger(__name__)
 
 
 class ZuulDFile(YAMLFile):
     """Representation of a YAML file that meets the Zuul.D schema.
     """
-    SCHEMA = File('_data/schemas/zuuld.json')
+    SCHEMA = URL('https://json.schemastore.org/zuul.json')
```

Comment:
I'm thinking about the unlikely event that this url is not accessible, cibyl should not fail in that case. Would there be a simple way to keep the old json schema and use it as fallback? It's very unlikely, so I think it only make sense if it's a simple change.

Reply:
In that case, this line: `tools.validators.from_remote(ZuulDFile.SCHEMA)` will raise a `DownloadError`. I could try-catch it and go without a schema for those cases. Schemas are ignored if `None` is passed on that argument.

I would go against having to maintain my own schema, as I have already faced cases that were valid but I did not contemplate correctly. I moved to this one in order to avoid so.

Reply:
That's fair enough, try-catching should be fine.

Reply:
Done.